app/controllers/sessions_controller.rb

```diff
@@ -26,7 +26,7 @@ def load_user
   def check_activated
     if @user.activated?
       log_in @user
-      params[:session][:remember_me] == Settings.user.remember ? remember(@user) : forget(@user)
+      params[:session][:remember_me].to_i == Settings.user.remember ? remember(@user) : forget(@user)
```

Comment:
Cái này tồn đọng ở chapter 9. Sau khi em Setting cho nó. Mà setting dùng số 1 kiểu int. params[:session][:remember_me] nó get được số 1 kiểu string. Nên so sánh nó luôn false.Mấy bữa login remmeber nó không lưu được cookie mà em không để ý. Giờ phát hiện ra nên em ép kiểu cho nó.

Reply:
sao em ko  để `Settings.user.remember` là kiểu string mà phải ép param sang kiểu int vậy?/
cái nào dễ thì làm chứ.haha

Reply:
Vì em muốn file Setting toàn số kiểu int luôn cho đẹp @Vu

Reply:
Tát. Câu trả lời vớ vẩn. Ép kiểu sẽ làm ctrinh xly chậm hơn á.